camel/agents/tool_agents/hugging_face_tool_agent.py

```diff
@@ -47,7 +47,7 @@ def __init__(
             raise ValueError(
                 "Could not import transformers tool agents. "
                 "Please setup the environment with "
-                "pip install huggingface_hub==0.14.1 transformers==4.29.0 diffusers accelerate datasets torch soundfile sentencepiece opencv-python"
+                "pip install huggingface_hub==0.14.1 transformers==4.31.0 diffusers accelerate datasets torch soundfile sentencepiece opencv-python"
```

Comment:
In the `try` clause prease add a runtime check for the version of transformers >= 4.31.0